src/language/languages.json

```diff
@@ -61,19 +61,26 @@
         "fileExtensions": ["md", "markdown", "mdown", "mkdn"],
         "blockComment": ["<!--", "-->"]
     },
-  
+
     "image": {
         "name": "Image",
         "fileExtensions": ["gif", "png", "jpe", "jpeg", "jpg", "ico", "bmp", "webp"],
         "isBinary": true
     },
-    
-    "audio": {
-        "name": "Audio",
-        "fileExtensions": ["mp3", "wav", "aif", "aiff", "ogg"],
+
+    "video": {
```

Comment:
See my comment above about using `Content`.  Let's use the types listed in https://github.com/mozilla/brackets/blob/master/src/filesystem/impls/filer/lib/content.js#L93-L108 for what we accept.